ui/lib/core/addon/templates/components/confirm/trigger.hbs

```diff
@@ -0,0 +1,41 @@
+{{#if showConfirm}}
+  {{#maybe-in-element wormholeReference false}}
+    <div class="confirm-action-message">
+      <div class="message is-highlight">
+        <div class="message-title">
+          <Icon @glyph="alert-triangle" />
+          {{title}}
+        </div>
+        <p>
+          {{message}}
+        </p>
+      </div>
+      <div class="confirm-action-options">
+        <button
+          type="button"
+          disabled={{unless renderedTrigger true}}
+          class="link is-destroy"
+          data-test-confirm-button="true"
+          onclick={{action onConfirm}}>
+          {{confirmButtonText}}
+        </button>
+        <button
+          type="button"
+          disabled={{unless renderedTrigger true}}
+          class="link"
+          data-test-confirm-cancel-button="true"
+          {{action this.onCancel}}>
+          {{cancelButtonText}}
+        </button>
+      </div>
+    </div>
+  {{/maybe-in-element}}
+{{else}}
+  <button
+    type="button"
+    class="link is-destroy"
+    onclick={{onTrigger}}
```

Comment:
Mentioned this before, but I think this could be passed from confirm, or even just hardcoded here to set openTrigger to the trigger's id.

Reply:
looked into hardcoding `openTrigger` here but realized we still need to define and call `onTrigger` from the `Confirm` component because we need to update `openTrigger` and update the height so that the `confirm-wrapper` shrinks/grows according to whether a trigger is opened or not. thanks for the suggestion!